com.unity.render-pipelines.high-definition/Documentation~/Override-Physically-Based-Sky.md

```diff
@@ -133,6 +133,8 @@ The default values in either mode make it so the planet's surface is at **0** on
 ## Warmup cost
 
 This sky type requires heavy precomputation to be rendered. Because of this, the first few frames (depending on the *Number of bounces* parameter) are going to take much longer. This needs to be taken into consideration when switching from another sky type to the Physically Based Sky for example as it might produce a noticeable drop in framerate.
```

Comment:
Improved this bit too. Suggested change: 

When you use a Physically Based Sky it might cause a noticeable drop in framerate. This is because HDRP performs a large amount of precomputations to render a Physically Based Sky, so the first few frames (depending on the **Number of bounces** parameter) takes more time to render than other HDRP sky types.

Reply:
Change sounds good but I think it's missing emphasis on this part:
> This needs to be taken into consideration when switching from another sky type to the Physically Based Sky for >example as it might produce a noticeable drop in framerate.
That stresses out the fact that switching sky type during gameplay is going to show this drop in framerate.

Reply:
Thanks for your feedback! I've changed my suggestion to the following: 

When you switch to or from a Physically Based Sky, it might cause a noticeable drop in framerate. This is because HDRP performs a large amount of precomputations to render a Physically Based Sky, so the first few frames (depending on the **Number of bounces** parameter) takes more time to render than other HDRP sky types.